templates/_lib.hcl

```diff
@@ -1,9 +1,69 @@
-{% macro migration_reschedule() %}
+{%- macro migration_reschedule() %}
   reschedule {
     attempts = 10
     interval = "5m"
     delay = "5s"
     delay_function = "exponential"
     max_delay = "30s"
   }
-{% endmacro %}
+{%- endmacro %}
+
+{%- macro authproxy(name, host, upstream) %}
```

Comment:
Maybe rename this to `authproxy_group(..)` to keep the naming scheme?

Reply:
Yup great idea thanks :+1: